### PR TITLE
rakudo-star: update to 2015.09

### PR DESCRIPTION
**Pull request by xaduha**

### CODE REVIEW

File: srcpkgs/rakudo-star/template [hunk 1]
```diff
@@ -13,8 +13,13 @@ maintainer="Ruslan <axetwe@gmail.com>"
 homepage="http://rakudo.org"
 license="Artistic-2.0"
 distfiles="http://rakudo.org/downloads/star/rakudo-star-${version}.tar.gz"
-checksum="84d7812a735eedc39d7c0898d4fd15ecd82563971744b2bc6ff0a1c581c82910"
-depends="MoarVM nqp"
+checksum="99b0332c4a05d444876efff58714104fc3cbf5f875174c1e410bedca03a6880d"
+depends="MoarVM>=${version} nqp>=${version}"
+
+pre_install() {
+	mkdir -p /usr/lib/perl6/share/perl6/runtime
```
Comment (xtraeme) [inline srcpkgs/rakudo-star/template]: You are creating a directory and a symlink in the masterdir, is this really what you wanted to achieve?

**Comment by xaduha** [comment]
If it doesn't interfere with anything, then yes. It's a quick and dirty workaround, there's module precompilation stage that is a part of 'make install', path to that file gets encoded in shell files.

**Comment by xaduha** [comment]
@xtraeme is there something else needed?

File: srcpkgs/rakudo-star/template [hunk 2]
```diff
@@ -13,9 +13,16 @@ maintainer="Ruslan <axetwe@gmail.com>"
...
+	ln -s /destdir/rakudo-star-${version}/usr/lib/perl6/share/perl6/runtime/perl6.moarvm /usr/lib/perl6/share/perl6/runtime/perl6.moarvm
+}
 
 post_install() {
 	vinstall ${FILESDIR}/rakudo-star.sh 644 etc/profile.d
+	rm    /usr/lib/perl6/share/perl6/runtime/perl6.moarvm
```
Comment (xtraeme) [inline srcpkgs/rakudo-star/template]: Please change this cleanup to `do_clean()` instead.

**Comment by xtraeme** [comment]
Looks good if you make the cleanup in `do_clean()`.

Thanks :+1:

**Comment by xaduha** [comment]
@xtraeme done, moved to do_clean()

**Pull request merged**